Backpack/CardButton/Classes/CardButtonSize.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@objc
```

Comment:
I dont think we need to support Objective-C, lets drop this and make the enum not extend from UInt

Reply:
sure!

Reply:
updated in [b68bdd7](https://github.com/Skyscanner/backpack-ios/pull/1564/commits/b68bdd798406bb4a4cb79c1117778d7c708ed09f)